src/emulator/modules/SceGxm/SceGxm.cpp

```diff
@@ -1507,30 +1532,24 @@ EXPORT(int, sceGxmTerminate) {
     return 0;
 }
 
-EXPORT(int, sceGxmTextureGetData, const emu::SceGxmTexture *texture) {
-    assert(texture != nullptr);
```

Comment:
I think we should keep these asserts for easier debugging. If real Vitas would crash thats OK if we assert too.

Reply:
Re-added